### ISSUE TITLE
Shears and lighter durability

### ISSUE THREAD

**Issue by deathcap** [deathcap/voxel-pumpkin#2]
Currently, shears can be reused indefinitely; perhaps they should have finite tool durability (like [voxel-pickaxe](https://github.com/deathcap/voxel-pickaxe))

**Comment by z3t0** [comment]
agreed, I think they should be incorporated into voxel-pickaxe (which should be renamed to voxel-item-tools)

**Comment by deathcap** [comment]
Maybe factor out into a new plugin like "voxel-shears" (better to keep smaller moduler separate plugins), but it could potentially reuse some common code with voxel-pickaxe

**Comment by z3t0** [comment]
@deathcap Oh okay, that makes sense
